daemon/cluster/convert/swarm.go

```diff
@@ -12,6 +14,15 @@ import (
 
 // SwarmFromGRPC converts a grpc Cluster to a Swarm.
 func SwarmFromGRPC(c swarmapi.Cluster) types.Swarm {
+	var defaultAddrPool []*net.IPNet
+	if c.DefaultAddressPool != nil {
+		for _, address := range c.DefaultAddressPool {
+			_, b, err := net.ParseCIDR(address)
+			if err == nil {
```

Comment:
what happens on an error ? Is it ignored silently ?

Reply:
This information is used  only for display purpose. so if there is a error reading CIDR then we ignore it and proceed further.

Reply:
I don't think that the `if c.DefaultAddressPool {` is really necessary.  It is legal to try to `range` over a nil list.

Reply:
will remove it